source/operator.rst

```diff
@@ -79,6 +79,10 @@ main subsystems of the cluster:
      - ``SmartUpdate``
      - A strategy the Operator uses for :ref:`upgrades<operator-update>`
 
+   * - autoPXCRecovery
```

Comment:
If I see correctly this is named differently and it goes under `pxc` section. The option is called `autoRecovery: true`